src/main.rs

```diff
@@ -40,41 +40,53 @@ fn main() -> BoxResult<()> {
 async fn async_main() -> BoxResult<()> {
     dotenv::dotenv().ok();
     println!("Booting ingestion server...");
-    let mongo_url = env::var("MONGO_URL")?;
+    let mongo_url = match env::var("MONGO_URL") {
+        Ok(url) => url,
+        _ => panic!("MONGO_URL variable not specified. Please specify the MongoDB's URL in this format: mongodb://user:password@some_mongo"),
+    };
 
     let mut client_options = ClientOptions::parse(mongo_url).await?;
     client_options.app_name = Some("Sentinel Surveillance".to_string());
-    client_options.tls = Some(Tls::Enabled(Default::default()));
+    match env::var("TLS") {
+        Ok(tls) => match tls.as_str() {
+            "true" => client_options.tls = Some(Tls::Enabled(Default::default())),
+            _ => println!("TLS Disabled"),
+        },
+        _ => println!("TLS Disabled"),
+    }
 
     let client = Client::with_options(client_options)?;
 
-    let database = client.database("imagine2022");
-
+    let database = match env::var("DATABASE") {
+        Ok(db_name) => client.database(&db_name),
+        _ => panic!("DATABASE variable not specified. Please specify DATABASE name."),
```

Comment:
If you match only returns itself or panics, then I think the `expect` method is better. Same behavior. Less code. More conventional. Same here and above.

Reply:
Oops, I totally forgot about `expect`.